booking-engine-api/guidelines/authentication.md

```diff
@@ -0,0 +1,36 @@
+# Authentication
+
+The __Mews Booking Engine API__ is a public API that requires client authentication and authorization. 
+
+## Client authentication
+
+Authentication is the process of verifying the identity of the client. To access the API, you must identify your client application by providing the `Client` property in all requests. This ensures that your client application is recognized and allowed to interact with the API.
+
+### Pre-registered
+
+Your client application needs to be pre-registered with Mews Support. You can open a ticket from __Mews Operations__ via the Mews Digital Assistant. The registration request should contain:
```

Comment:
question: Do you think we could link to some help article for Mews Digital Assistant?

Reply:
Yes, I agree. I will add a link to Help Article [How to use the Mews Digital Assistant in Mews Operations](https://help.mews.com/s/article/How-to-use-the-Mews-Digital-Assistant?language=en_US).